common/src/main/java/org/apache/rocketmq/common/BrokerConfig.java

```diff
@@ -187,6 +187,8 @@
 
     private boolean autoDeleteUnusedStats = false;
 
+    private boolean isolateLogEnable = false;
```

Comment:
It would be better to add some comments to explain the meaning of this parameter

Reply:
done